homeassistant/components/deconz/device_trigger.py

```diff
@@ -127,6 +127,26 @@
     (CONF_LONG_RELEASE, CONF_BOTTOM_BUTTONS): {CONF_EVENT: 6003},
 }
 
+FELLER_EDIZIODUE_FRIENDS_OF_HUE_MODEL = "Feller EDIZIOdue"
+FELLER_EDIZIODUE_FRIENDS_OF_HUE = {
+    (CONF_SHORT_PRESS, CONF_BUTTON_1): {CONF_EVENT: 1000},
+    (CONF_SHORT_RELEASE, CONF_BUTTON_1): {CONF_EVENT: 1002},
+    (CONF_LONG_PRESS, CONF_BUTTON_1): {CONF_EVENT: 1001},
+    (CONF_LONG_RELEASE, CONF_BUTTON_1): {CONF_EVENT: 1003},
+    (CONF_SHORT_PRESS, CONF_BUTTON_2): {CONF_EVENT: 2000},
+    (CONF_SHORT_RELEASE, CONF_BUTTON_2): {CONF_EVENT: 2002},
+    (CONF_LONG_PRESS, CONF_BUTTON_2): {CONF_EVENT: 2001},
+    (CONF_LONG_RELEASE, CONF_BUTTON_2): {CONF_EVENT: 2003},
+    (CONF_SHORT_PRESS, CONF_BUTTON_3): {CONF_EVENT: 3000},
+    (CONF_SHORT_RELEASE, CONF_BUTTON_3): {CONF_EVENT: 3002},
+    (CONF_LONG_PRESS, CONF_BUTTON_3): {CONF_EVENT: 3001},
+    (CONF_LONG_RELEASE, CONF_BUTTON_3): {CONF_EVENT: 3003},
+    (CONF_SHORT_PRESS, CONF_BUTTON_4): {CONF_EVENT: 4000},
+    (CONF_SHORT_RELEASE, CONF_BUTTON_4): {CONF_EVENT: 4002},
+    (CONF_LONG_PRESS, CONF_BUTTON_4): {CONF_EVENT: 4001},
+    (CONF_LONG_RELEASE, CONF_BUTTON_4): {CONF_EVENT: 4003},
+}
+
```

Comment:
Hi @bqstony 

This is exactly the same button mapping as the existing hue dimmer switch:

```python
HUE_DIMMER_REMOTE = {
    (CONF_SHORT_PRESS, CONF_TURN_ON): {CONF_EVENT: 1000},
    (CONF_SHORT_RELEASE, CONF_TURN_ON): {CONF_EVENT: 1002},
    (CONF_LONG_PRESS, CONF_TURN_ON): {CONF_EVENT: 1001},
    (CONF_LONG_RELEASE, CONF_TURN_ON): {CONF_EVENT: 1003},
    (CONF_SHORT_PRESS, CONF_DIM_UP): {CONF_EVENT: 2000},
    (CONF_SHORT_RELEASE, CONF_DIM_UP): {CONF_EVENT: 2002},
    (CONF_LONG_PRESS, CONF_DIM_UP): {CONF_EVENT: 2001},
    (CONF_LONG_RELEASE, CONF_DIM_UP): {CONF_EVENT: 2003},
    (CONF_SHORT_PRESS, CONF_DIM_DOWN): {CONF_EVENT: 3000},
    (CONF_SHORT_RELEASE, CONF_DIM_DOWN): {CONF_EVENT: 3002},
    (CONF_LONG_PRESS, CONF_DIM_DOWN): {CONF_EVENT: 3001},
    (CONF_LONG_RELEASE, CONF_DIM_DOWN): {CONF_EVENT: 3003},
    (CONF_SHORT_PRESS, CONF_TURN_OFF): {CONF_EVENT: 4000},
    (CONF_SHORT_RELEASE, CONF_TURN_OFF): {CONF_EVENT: 4002},
    (CONF_LONG_PRESS, CONF_TURN_OFF): {CONF_EVENT: 4001},
    (CONF_LONG_RELEASE, CONF_TURN_OFF): {CONF_EVENT: 4003},
}
```

No need to repeat it, just add the new model key `FELLER_EDIZIODUE_FRIENDS_OF_HUE_MODEL` like it's done for the 2 existing models for this mapping:

```python
REMOTES = {
    HUE_DIMMER_REMOTE_MODEL_GEN1: HUE_DIMMER_REMOTE,
    HUE_DIMMER_REMOTE_MODEL_GEN2: HUE_DIMMER_REMOTE,
    # ...
    FELLER_EDIZIODUE_FRIENDS_OF_HUE_MODEL: HUE_DIMMER_REMOTE,
}
```

Reply:
Oh, sorry, I see that this one is using the "button_1" to "button_4" keys instead on/off/dim u/down.

Forget what I just said :)